ukwa/npld-player
# Handle external links and new windows

(https://github.com/ukwa/npld-player/issues/12) Open links without specified prefix in default browser, and handle links that request a new window.

### Manual testing
1. Run test page server with `yarn start-test-server`. In a separate shell, run app dev server with `yarn start`
2. Click "Open webrecorder.net in app (target=_blank)". Verify link loads in current webview
3. Mac: Command-click "Open webrecorder.net in app". Verify link loads in current webview
4. Click "Open example.com in browser" and Open example.com in browser (target=_blank)". Verify both links open in default browser
